RouterTesting/RouterTesting/Controllers/MainViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MyViewController.swift
+//  RouterTesting
+
+import UIKit
+
+/// Основной Экран приложения
+final class MainViewController: UIViewController {
+    // MARK: - Private Constants
+   private enum Constants {
+        static let keyText = "isLogin"
+    }
+    
+    // MARK: - Private Properties
+    private var router: GoLoginRouter?
+
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        router = GoLoginRouter(controller: self)
+    }
+    
+    // MARK: - IBActions
```

Comment:
```suggestion
    // MARK: - Private IBActions
```
раз везде марки Private IBActions, то и тут добавь